.babelrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
```

Comment:
Can you change the indentation of this file to 2 spaces?

Reply:
👍 sure thing